docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -308,6 +308,21 @@ agent you want to unenroll.
 
 . Click **Force unenroll**. 
 
+[discrete]
+[[endpoint-not-uninstalled-with-agent]]
+== {elastic-endpoint} uninstallation fails
+
+When you uninstall {agent}, all the programs managed by {agent}, such as
+{elastic-endpoint}, are also removed. If for some reason uninstallation fails,
+it's possible that {elastic-endpoint} might remain on your system.
```

Comment:
I'm getting nit-picky since everything else looks good: `If for some reason {thing} fails, it's possible that {thing} might {do thing}` seems wordy to me. What about any of these?

```
If uninstallation fails, {elastic-endpoint} might remain on your system.
```

Or perhaps the first "for some reason" is necessary since this is a troubleshooting topic:

```
If for some reason uninstallation fails, {elastic-endpoint} might remain on your system.
```

🤷 I needed something to comment on 😉

Reply:
haha...totally valid. I get wordy in the evening. Must be all the wine. hahaha.